openapi/platform/platform.yaml

```diff
@@ -401,6 +401,322 @@ paths:
               properties:
                 data:
                   $ref: '#/components/schemas/businessLocations'
+  /users:
```

Comment:
The server has a bunch of extra endpoints that are not yet ready to be displayed in the public documentation. Could you limit the changes in this file to only the ones related to automations? Let me know if you want me to do it.